packages/visx-annotation/src/components/Label.tsx

```diff
@@ -15,6 +15,8 @@ export type LabelProps = {
   backgroundPadding?: number | { top?: number; right?: number; bottom?: number; left?: number };
   /** Additional props to be passed to background SVGRectElement. */
   backgroundProps?: React.SVGProps<SVGRectElement>;
+  /** Pass in a custom element as the label to style as you like. Renders inside a <foreignObject> */
```

Comment:
nit – period for docs.

```suggestion
  /** Pass in a custom element as the label to style as you like. Renders inside a <foreignObject>. */
```

I wonder also if it'd be worth noting from the [original issue](https://github.com/airbnb/visx/issues/1173#issuecomment-1014380545) something like 

> Note that copy/pasting content from a browser may not work if you use this option because not all non-browser SVG renderers render HTML `<foreignObject>`s

I'm actually not sure if this is true since the DOM tree in the implementation has a div `<foreignObject><div>...` so maybe don't have to worry about it for now. cc @richsilv